core/src/com/mygdx/game/MyGdxGame.java

```diff
@@ -2,32 +2,119 @@
 
 import com.badlogic.gdx.ApplicationAdapter;
 import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
 import com.badlogic.gdx.graphics.GL20;
 import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Animation;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
 
 public class MyGdxGame extends ApplicationAdapter {
 	SpriteBatch batch;
-	Texture img;
-	
+	Texture m;
+	TextureRegion stand, flip;
+	float x, y, xv, yv;
+	boolean canFlip, faceRight = true;
+	Animation walk;
+	float time;
+	static final float MAX_VELOCITY = 500;
+	static final int WIDTH = 16;
+	static final int HEIGHT = 16;
+	static final int DRAW_WIDTH = WIDTH * 5;
+	static final int DRAW_HEIGHT = HEIGHT * 5;
+	static final float MAX_JUMP_VELOCITY = 2000;
+	static final int GRAVITY = -50;
+
+	TextureRegion down, up, right, left;
+
+
+
 	@Override
-	public void create () {
+	public void create() {
 		batch = new SpriteBatch();
-		img = new Texture("badlogic.jpg");
+		m = new Texture("tiles.png");
+		Texture tiles = new Texture("tiles.png");
+		TextureRegion[][] grid = TextureRegion.split(tiles, 16, 16);
+		down = grid[6][0];
+		up = grid[6][1];
+		right = grid[6][3];
+		left = new TextureRegion(right);
+		left.flip(true, false);
+		walk = new Animation(0.2f, grid [6][0], grid[6][1], grid[6][3]);
 	}
 
 	@Override
-	public void render () {
-		Gdx.gl.glClearColor(1, 0, 0, 1);
-		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
-		batch.begin();
-		batch.draw(img, 0, 0);
-		batch.end();
+	public void render() {
+
+		time += Gdx.graphics.getDeltaTime();
+
+		move();
+
+		TextureRegion img;
+
+//		if (y > 0) {
+//			img = flip;
+//		} else if (xv != 0) {
+//			img = walk.getKeyFrame(time, true);
+//		} else {
+//			img = stand;
+
+			Gdx.gl.glClearColor(0, 2, 0, 1);
+			Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+			batch.begin();
+			if (faceRight) {
+				batch.draw(right, x, y, DRAW_WIDTH, DRAW_HEIGHT);
+			} else {
+				batch.draw(left, y, x,  DRAW_WIDTH  , DRAW_HEIGHT);
+			}
```

Comment:
Check line 68.  are you sure that's what you want.  You also need to add face up and down.